scripts/operator/push-operator-to-snyk-upstream.sh

```diff
@@ -18,7 +18,7 @@ CURRENT_DIRECTORY=$1
 COMMUNITY_FOLDER_LOCATION=$2
 COMMUNITY_OPERATORS_UPSTREAM_LOCATION="${CURRENT_DIRECTORY}/community-operators"
 DEPLOY_LOCATION="${COMMUNITY_OPERATORS_UPSTREAM_LOCATION}/${COMMUNITY_FOLDER_LOCATION}"
-OPERATOR_LOCATION="${CURRENT_DIRECTORY}/snyk-operator/deploy/olm-catalog/snyk-operator"
+OPERATOR_LOCATION="${CURRENT_DIRECTORY}/snyk-operator/deploy/olm-catalog/snyk-operator/${OPERATOR_PATH}"
```

Comment:
I think this should be `OPERATOR_LOCATION="${OPERATOR_PATH}"` otherwise it will be assigned something like this: `${CURRENT_DIRECTORY}/snyk-operator/deploy/olm-catalog/snyk-operator/var/folders/x2/5bsy5xqx4b31j4gpvzyfhl_40000gn/T/tmpz6luzr9u` and it will error out again! The `package_operator` script returns an absolute path

Reply:
you right will fix it :)